Move collector docs into reference section (#3630)
OpenNMS/opennms: docs/modules/operation/pages/thresholds/thresholding.adoc

[[metric-collect]]
=== Determine you are collecting metric
In this case, we have chosen a metric (number of logged-in users) that is collected by default.

> We are also using data collected via SNMP. (For information on other collectors, see <<performance-data-collection/collectors.adoc#collectors-overview,Collectors>>.)

Reviewer: ```suggestion
We are also using data collected via SNMP. 
(For information on other collectors, see <<performance-data-collection/collectors.adoc#collectors-overview,Collectors>>.)
```
separate line

[[metric-collect]]
=== Determine you are collecting metric
In this case, we have chosen a metric (number of logged-in users) that is collected by default.

> We are also using data collected via SNMP. 

Reviewer 2: ```suggestion
We are also using data collected via SNMP.
```

> (For information on other collectors, see <<performance-data-collection/collectors.adoc#collectors-overview,Collectors>>.)

Reviewer 2: Send them directly to the reference section instead of the redirect stub.
```suggestion
(For information on other collectors, see xref:reference:configuration/performance-data-collection/introduction.adoc[Collectors].)
```
